test/built-ins/DataView/prototype/getFloat16/not-a-constructor.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2024 Kevin Gibbons. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-ecmascript-standard-built-in-objects
+description: >
+  DataView.prototype.getFloat16 does not implement [[Construct]], is not new-able
+includes: [isConstructor.js]
+features: [Float16Array, Reflect.construct, DataView, arrow-function, ArrayBuffer]
+---*/
+
+assert.sameValue(
+  isConstructor(DataView.prototype.getFloat16),
+  false,
+  'isConstructor(DataView.prototype.getFloat16) must return false'
+);
+
+assert.throws(TypeError, () => {
+  let dv = new DataView(new ArrayBuffer(16)); new dv.getFloat16(0, 0);
+}, '`let dv = new DataView(new ArrayBuffer(16)); new dv.getFloat16(0, 0)` throws TypeError');
```

Comment:
Nitpick: I know the other not-a-constructor tests have these autogenerated assert messages, but let's not add more :stuck_out_tongue_closed_eyes:
Could I suggest "DataView.prototype.getFloat16 should not be a constructor" and "Constructing DataView.prototype.getFloat16 should throw TypeError"?

Reply:
Sure. Would you also like me to update the tests I copied this from?

Reply:
Opened https://github.com/tc39/test262/pull/4024.